BUILD.md

```diff
@@ -48,52 +49,14 @@ written in Go. The dashboard UI is a React application.
 
 - [`linkerd2-proxy`](https://github.com/linkerd/linkerd2-proxy): Rust source
   code for the proxy lives in the linkerd2-proxy repo.
-- [`linkerd2-proxy-api`](https://github.com/linkerd/linkerd2-proxy-api): Protobuf
-  definitions for the data plane APIs live in the linkerd2-proxy-api repo.
+- [`linkerd2-proxy-api`](https://github.com/linkerd/linkerd2-proxy-api):
+  Protobuf definitions for the data plane APIs live in the linkerd2-proxy-api
+  repo.
 
 ## Components
 
 ![Linkerd2 Components](https://g.gravizo.com/source/svg/linkerd2_components?https%3A%2F%2Fraw.githubusercontent.com%2Flinkerd%2Flinkerd2%2Fmaster%2FBUILD.md)
 
-<details>
```

Comment:
Did you mean to strip the graphviz out?

Reply:
I did originally. We talked about this in private channel, but it is still necessary. It was rendering without it only because the link is hard coded for the master branch.

I added it back, but had to disable some markdown rules in this file because it is now a mix of indented and fenced code block styles. I can't fence (`'''html`) this part because then it actually renders it as an HTML code block which we don't want.

Reply:
I did originally. We talked about this in private channel, but it is still necessary. It was rendering without it only because the link is hard coded for the master branch.

I added it back, but had to disable some markdown rules in this file because it is now a mix of indented and fenced code block styles. I can't fence (`'''html`) this part because then it actually renders it as an HTML code block which we don't want.